tests/integration/clientv3/experimental/recipes/v3_barrier_test.go

```diff
@@ -46,6 +47,11 @@ func testBarrier(t *testing.T, waiters int, chooseClient func() *clientv3.Client
 		t.Fatalf("able to double-hold barrier")
 	}
 
+	// put any key just used to move revision forward
```

Comment:
nits: "put a random key to move the revision forward"

Reply:
> nits: "put a random key to move the revision forward"

fixed, thx!